PebbleSharp.Mono.BlueZ5/obj/Debug/PebbleSharp.Mono.BlueZ5.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,5 @@
+C:\Users\brook_000\Desktop\PebbleSharp\PebbleSharp\PebbleSharp.Mono.BlueZ5\bin\Debug\PebbleSharp.Mono.BlueZ5.dll
+C:\Users\brook_000\Desktop\PebbleSharp\PebbleSharp\PebbleSharp.Mono.BlueZ5\bin\Debug\PebbleSharp.Mono.BlueZ5.pdb
+C:\Users\brook_000\Desktop\PebbleSharp\PebbleSharp\PebbleSharp.Mono.BlueZ5\obj\Debug\PebbleSharp.Mono.BlueZ5.csprojResolveAssemblyReference.cache
+C:\Users\brook_000\Desktop\PebbleSharp\PebbleSharp\PebbleSharp.Mono.BlueZ5\obj\Debug\PebbleSharp.Mono.BlueZ5.dll
+C:\Users\brook_000\Desktop\PebbleSharp\PebbleSharp\PebbleSharp.Mono.BlueZ5\obj\Debug\PebbleSharp.Mono.BlueZ5.pdb
```

Comment:
I did not mean to include those.  Do you mind if we update the .gitignore to one of the vs/c# ones from github?